contracts/external/cw-payroll-factory/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name ="cw-payroll-factory"
+version = "2.0.0-beta"
+authors = ["Jake Hartnell"]
```

Comment:
@0xekez / @blue-note feel free to add your names as well! Thanks for the commits! :heart:

Reply:
Added.